pkg/logging/logging.go

```diff
@@ -0,0 +1,61 @@
+// package logging is a small wrapper around github.com/k8snetworkplumbingwg/cni-log
+
+package logging
+
+import (
+	cnilog "github.com/k8snetworkplumbingwg/cni-log"
+)
+
+var (
+	logLevelDefault = cnilog.ErrorLevel
+)
+
+func init() {
+	SetLogFile("")
+	SetLogLevel("")
+}
+
+// SetLogLevel sets the log level to either verbose, debug, info, warn, error or panic. If an invalid string is
+// provided, it uses error.
+func SetLogLevel(l string) {
+	ll := cnilog.StringToLevel(l)
+	if ll == cnilog.InvalidLevel {
+		ll = logLevelDefault
+	}
+	cnilog.SetLogLevel(ll)
+}
+
+// SetLogFile sets the log file for logging. If the empty string is provided, it uses stderr.
+func SetLogFile(fileName string) {
+	if fileName == "" {
+		cnilog.SetLogStderr(true)
+		return
+	}
+	cnilog.SetLogStderr(false)
```

Comment:
to remove and error message `cni-log: filename is required when logging to stderr is off - will not log anything`
please switch between this 2 lines

meaning first put the setLogFile and then the setLogStderr